.github/workflows/ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     needs: build-operator
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
     env:
-      IMG: ghcr.io/dragonflydb/operator:${{ github.ref_name }}
+      IMG: ghcr.io/${{ github.actor }}/dragonfly-operator:${{ github.ref_name }}
```

Comment:
Maybe I think this is more conducive to the development of the fork project. If you use the operator image name, such as monlor/operator, it will be easy to confuse.

Reply:
But it is not needed for the `release-operator` job. This job runs only when a new version is released, not when a pull request is created.

Reply:
Okay, I restored the image name.

Reply:
You need to revert this line (and consecutive lines containing `ghcr.io/${{ github.actor }}/operator`) in order to make it work.